CONTRIBUTING.md

```diff
@@ -0,0 +1,25 @@
+# Contributing
+
+We accept contributions via Pull Requests on [GitHub](https://github.com/Yoast/YoastSEO.js).
+
+## Pull Requests
+
+- **Coding standards** - The easiest way to verify the coding standards is to use `grunt eslint`.
+
+- **Add tests!** - Your patch won't be accepted if it doesn't have tests.
+
+- **Document any change in behaviour** - Make sure the `README.md` and any other relevant documentation are kept up-to-date.
+
+- **Create feature branches** - Don't ask us to pull from your master branch.
```

Comment:
Maybe we can ditch this requirement? Isn't its primary usecase already covered by the One PR per feature requirement?